packages/game/src/lib/player.ts

```diff
@@ -1,35 +1,23 @@
 import { ReadonlyDeep } from 'type-fest';
 import { Card, Hand } from './common-types';
 
-export type PlayerLabel = typeof Player.One | typeof Player.Two;
+export enum PlayerLabel {
+  One = 'one',
+  Two = 'two',
+}
 
-export type PlayerOptions = {
+export type Player = {
   label: PlayerLabel;
-  hand: ReadonlyDeep<Hand>;
+  hand: Hand;
 };
 
-export class Player {
-  static One = 'one' as const;
-  static Two = 'two' as const;
-
-  #hand: Hand;
-
-  public readonly label: PlayerLabel;
-
-  constructor({ label, hand }: PlayerOptions) {
-    this.label = label;
-    this.#hand = structuredClone(hand) as Hand;
-  }
-
-  public get hand(): Readonly<Hand> {
-    return this.#hand as Readonly<Hand>;
-  }
+export function removeCard(hand: ReadonlyDeep<Hand>, card: Card): Hand {
+  const idx = hand.findIndex(
+    (_card) => _card?.name?.toLowerCase() == card.name.toLowerCase(),
+  );
 
-  public removeCard(card: Card) {
-    const idx = this.#hand.findIndex(
-      (_card) => _card?.name?.toLowerCase() == card.name.toLowerCase(),
-    );
+  const temp: Hand = [...hand];
+  temp[idx] = undefined;
 
-    this.#hand[idx] = undefined;
-  }
+  return temp;
```

Comment:
Maybe we call this `newHand` instead of `temp`

Reply:
Fixed.